connectors/src/logger/logger.ts

```diff
@@ -2,6 +2,15 @@ import pino, { LoggerOptions } from "pino";
 
 const NODE_ENV = process.env.NODE_ENV;
 const defaultPinoOptions: LoggerOptions = {
+  serializers: {
+    err: pino.stdSerializers.err,
```

Comment:
why do we treat differently err and error ?

Reply:
thanks @fontanierh . I followed chatGPT recommendation but this simpler version works too.